distributions/aws/examples/rds-s3/README.md

```diff
@@ -58,10 +80,10 @@ eksctl create cluster \
 You must make sure you have an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for your cluster and that it was added from `eksctl` >= `0.56` or if you already have an OIDC provider in place, then you must make sure you have the tag `alpha.eksctl.io/cluster-name` with the cluster name as its value. If you don't have the tag, you can add it via the AWS Console by navigating to IAM->Identity providers->Your OIDC->Tags.
 
 ## 2.0 Setup RDS, S3 and configure Secrets
-There are two ways to create the RDS and S3 resources before you deploy the Kubeflow manifests. Either use the automated python script we have provided by following the steps in section 2.1 or fall back on the manual setup steps as in section 2.2 below - 
+To facilitate creating the RDS and S3 resources an automated python script has been provided. 
+
+:warning: The automated python script setups an environment to be used for development/testing purposes. To help setup an environment that meets your production needs, please refer to the [RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html) and [S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide//Welcome.html) documentation. 
```

Comment:
> The automated python script setups an environment to be used for development/testing purposes  
  
Could you explain how-so?  
From my understanding the doc on https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql was making this warning because of the lack of Multi-AZ, automatic backup and unsecure default db credentials but now with the automated script users have Multi-AZ enabled by default, regular automatic backup and strong default db credentials (password generated using secrets manager etc).  
![image](https://user-images.githubusercontent.com/26939775/156234788-b535dedd-90c0-4c43-a5b3-14af691377a3.png)

Reply:
I see, I assumed the warning was also there as some sort of "clause of non-liability", although looking at this repository's license as well as the various best practices implemented by the automated setup script I should be able to remove this warning.